syft/pkg/alpm_metadata.go

```diff
@@ -0,0 +1,83 @@
+package pkg
+
+import (
+	"sort"
+	"time"
+
+	"github.com/anchore/packageurl-go"
+	"github.com/anchore/syft/syft/file"
+	"github.com/anchore/syft/syft/linux"
+	"github.com/scylladb/go-set/strset"
+)
+
+const AlpmDBGlob = "**/var/lib/pacman/local/**/desc"
```

Comment:
@wagoodman @Foxboron - I updated this to be more inline with our other DB glob matchers:
https://github.com/anchore/syft/blob/f35a252ecf923f410112eafdfd331119ed42edeb/syft/pkg/apk_metadata.go#L12-L39

This way Alpm packages are picked up in our integration tests now for the `pkg-coverage` image where `/var` is not at the root level.

Reply:
This makes sense :) Thanks!